src/docker-osx-dev

```diff
@@ -379,13 +384,63 @@ function init_boot2docker {
     log_info "Starting Boot2Docker VM"
     boot2docker start --vbox-share=disable
   fi
+
+  configure_boot2docker_vars
+}
+
+################################################################################
+# docker-machine manipulation
+################################################################################
+
+function get_json_val () { 
+    python -c "import json,sys;sys.stdout.write(json.dumps(json.load(sys.stdin)$1))" | sed -e 's/^"//'  -e 's/"$//'
```

Comment:
I'm a bit concerned about adding a dependency on python. Any alternative way to do this?